### DISCUSSION TITLE
How to test if my RCP coprocessor (nrf52840-dongle) is correctly programmed?

### DISCUSSION THREAD

**Discussion by olalonde** [openthread/openthread#10539] [Q&A]
It shows up under lsusb and I can connect to it via Nordic's serial terminal:

```sh
*** Booting nRF Connect SDK v2.7.0-5cb85570ca43 ***
*** Using Zephyr OS v3.6.99-100befc70c74 ***
~pt~
```

Is there any quick way to check that it works correctly?

**Comment by jwhui** [comment]
If you already have an OpenThread host configured (e.g. `ot-br-posix`), the easiest would be to connect the USB dongle and observe the `otbr-agent` logs.
